flask_excel/__init__.py

```diff
@@ -18,20 +18,37 @@ def get_file_tuple(self, field_name):
         filename = filehandle.filename
         extension = filename.split(".")[1]
         return extension, filehandle
-    
 
 Flask.request_class = ExcelRequest
 webio.ExcelResponse = Response
 
-from pyexcel_webio import (
-    make_response,
-    make_response_from_array,
-    make_response_from_dict,
-    make_response_from_records,
-    make_response_from_book_dict,
-    make_response_from_a_table,
-    make_response_from_query_sets,
-    make_response_from_tables
-)
-
-__VERSION__ = '0.0.3'
+def add_file_name(response, file_name, file_type):
```

Comment:
I agree.  A decorator could be provided if the file name wont change based on the request.  For example:

    @download_file_name("exported_data")
    @app.route("/download")
    def download():
      return excel.make_response(....

But itd be beneficial to allow the file name arg to vary based on the request.  A decorator could be applied to the proxied pyexcel methods, too.